src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -8,17 +8,28 @@
 [SuppressMessage("ReSharper", "StringLiteralTypo", Justification = "Patch names aren't proper words")]
 public static class PatchMapGenerator
 {
+    public static WorldGenerationSettings WorldSettings = new();
```

Comment:
All the methods in this class are static, so this has to be static.

Reply:
Why are the settings static here? Shouldn't they be a generation run specific?

Reply:
As I said, everything in this class is static for some reason, so an instance variable wouldn’t work.

Although now that I think about it, there’s probably no need to store them as a member variable here since they’re passed in every time. I’ll change that to resolve this.

Reply:
If the design starts to look like each method needs to pass the current settings all around, I think converting this to a non-static class would start to make a lot of sense so that the settings can be passed just once to the constructor.